packages/devtools_app/lib/src/flutter/scaffold.dart

```diff
@@ -191,11 +192,18 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
     // Build the screens for each tab and wrap them in the appropriate styling
     final tabBodies = [
       for (var screen in widget.tabs)
-        Align(
-          alignment: Alignment.topLeft,
-          child: Padding(
-            padding: DevToolsScaffold.appPadding,
-            child: screen.build(context),
+        Padding(
```

Comment:
@pq 
Padding + Align should trigger a lint to use Container.

Reply:
Cool!  I could use some context...  Issue opened: https://github.com/dart-lang/linter/issues/2059.  Maybe you could elaborate there when you have a moment?  Thanks!

Reply:
When an `Align` widget has a child of `Padding` or vice versa, we should trigger a lint that suggests instead using Container:
```
Container(
  alignment: ...,
  padding: ...,
);

Reply:
Thanks @kenzieschmoll!